### PR TITLE
[harvest] Add GMUHK item harvester

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
Fixes https://jira.sng.sk/browse/WEBUMENIA-1649

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/Harvest/Mappers/AuthorityEventMapper.php [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-class AuthorityEventMapper extends AbstractModelMapper
+class AuthorityEventMapper extends AbstractMapper
 {
     protected $modelClass = AuthorityEvent::class;
 
-    public function mapEvent(array $row, $locale) {
+    public function mapEvent(array $row) {
         $event = $this->serialize($row['event']);
-        return $this->chooseTranslation($event, $locale);
+        return $this->chooseTranslation($event, 'sk');
```
Comment (eronisko) [inline app/Harvest/Mappers/AuthorityEventMapper.php:13]:
```suggestion
        return $this->chooseTranslation($event, config('app.locale'));
```

File: app/Harvest/Mappers/AuthorityRelationshipMapper.php [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
+class AuthorityRelationshipMapper extends AbstractMapper
 {
     protected $modelClass = AuthorityRelationship::class;
 
-    public function mapType(array $row, $locale) {
+    public function mapType(array $row) {
         if (isset($row['type'][0])) {
             $type = $row['type'][0];
-            return $this->chooseTranslation($type, $locale);
+            return $this->chooseTranslation($type, 'sk');
```
Comment (eronisko) [inline app/Harvest/Mappers/AuthorityRelationshipMapper.php:14]:
```suggestion
            return $this->chooseTranslation($type, config('app.locale'));
```

**Review by eronisko — COMMENTED** [review]
I've included a few code style suggestions for your consideration, none critical.

My only objection would be the organization of the new Harvester/Importer/Mapper classes under the `Gmuhk` namespace. I think that `App\Harvest\Harvesters\GmuhkItemHarvester` is a better fit than `App\Harvest\Gmuhk\Harvesters\ItemHarvester`: the proposed organization duplicates directories and IMHO makes it harder to find the right classes.

**Comment by rastislav-chynoransky** [comment]
> I've included a few code style suggestions for your consideration, none critical.
> 
> My only objection would be the organization of the new Harvester/Importer/Mapper classes under the `Gmuhk` namespace. I think that `App\Harvest\Harvesters\GmuhkItemHarvester` is a better fit than `App\Harvest\Gmuhk\Harvesters\ItemHarvester`: the proposed organization duplicates directories and IMHO makes it harder to find the right classes.

You're totally right, will do! Thanks also for your contribution

**Review by eronisko — APPROVED** [review]
Some failing tests but LGTM otherwise. Thanks.

**Pull request merged**